src/main/java/aquarium/shop/order/Starter.java

```diff
@@ -1,31 +1,47 @@
 package aquarium.shop.order;
 
+import aquarium.shop.basket.Basket;
 import aquarium.shop.product.Product;
+import aquarium.shop.product.ProductRepository;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
 import javax.annotation.PostConstruct;
 import java.math.BigDecimal;
-import java.util.HashMap;
 
 @Component
 public class Starter {
 
     @Autowired
-    private OrderRepository orderRepository;
+    private OrderService orderService;
+
+    @Autowired
+    private Basket basket;
+
+    @Autowired
+    private ProductRepository productRepository;
 
     @PostConstruct
     public void startUp() {
-        HashMap<Product, Integer> content = new HashMap<>();
-        BigDecimal wholePrice = BigDecimal.valueOf(0);
+
+
+
+        Product gupy = productRepository.save(new Product("animal", "gupy", 7.));
+        Product molly = productRepository.save(new Product("animal", "molly", 10.));
+        Product root = productRepository.save(new Product("equipment", "root", 30.));
+
+        basket.addProduct(gupy, 6);
+        basket.addProduct(molly);
+        basket.addProduct(root);
 
         Address address = new Address("Borownice", "12B", "33-350",
                 "Poland", "Piwniczna");
         OrderUser orderUser = new OrderUser("Jan", "Zytkowicz");
         DeliveryDetails deliveryDetails = new DeliveryDetails(address, orderUser, Shipment.DELIVERY);
-        OrderDetails order = new OrderDetails(deliveryDetails, PaymentStatus.NOT_PAID, OrderStatus.APPROVED, wholePrice);
+        OrderDetails order = new OrderDetails(deliveryDetails, PaymentStatus.NOT_PAID, OrderStatus.APPROVED );
```

Comment:
I think that we can implement that starter a little bit better to avoid using basket as whole flow is not object of that insertion. I will explain that offline and we can do it together.